acceptance-tests/helpers/brokers/create.go

```diff
@@ -63,8 +64,18 @@ func WithSourceDir(dir string) Option {
 		b.dir = dir
 	}
 }
+func WithConfig(config map[string]interface{}, dir string) {
```

Comment:
issue: use `any` rather than `interface{}` to be consistent with the rest of the code base.